default-recommendations/conditional-shortcuts.rkt

```diff
@@ -268,13 +268,28 @@
   (when condition body))
 
 
+(define-refactoring-rule explicit-cond-else-void
+  #:description "Add an explicit `[else (void)]` clause to make the default behavior clear."
+  #:literals (cond else void)
+  (cond-id:cond clause ...)
```

Comment:
If you use `~and` with `~not` here, you can bind the `clause` variable and simultaneously require that `else` isn't used, making the `#:when` condition on line 277 unnecessary.


```suggestion
  (cond-id:cond (~and clause (~not [else . _])) ...)
```